src/DynamoCore/Models/RunSettings.cs

```diff
@@ -16,15 +16,13 @@ public enum RunType { Manual, Automatic, Periodic }
     /// </summary>
     public class RunSettings : NotificationObject
     {
-        #region private members
+        #region Class Data Members and Properties
 
         private int runPeriod;
         private RunType runType;
         private bool runEnabled;
 
-        #endregion
-
-        #region properties
+        public const int DefaultRunPeriod = 1000;
```

Comment:
Presently the system is incredibly unstable at 100ms.  This must be increased until that is addressed.